contracts/multisig/src/contract.rs

```diff
@@ -25,28 +25,15 @@ mod query;
 
 const CONTRACT_NAME: &str = env!("CARGO_PKG_NAME");
 const CONTRACT_VERSION: &str = env!("CARGO_PKG_VERSION");
+const BASE_VERSION: &str = "1.0.0";
```

Comment:
this test will break post release, so needs to be fixed again

Reply:
that is kind of the point of these tests, we should make sure we have them everywhere. That way we don't forget to remove the migration